diff --git a/node_modules/@nx/next/plugins/with-nx.js b/node_modules/@nx/next/plugins/with-nx.js
index 0635d96..8d96223 100644
--- a/node_modules/@nx/next/plugins/with-nx.js
+++ b/node_modules/@nx/next/plugins/with-nx.js
@@ -235,8 +235,24 @@ function getNextConfig(nextConfig = {}, context = getWithNxContext()) {
                 {
                     test: /\.svg$/i,
                     issuer: /\.[jt]sx?$/,
-                    resourceQuery: { not: [/url/] },
-                    use: ['@svgr/webpack'],
+                    resourceQuery: { 
+                        not: [/url/] 
+                    },
+                    use: [{
+                        loader: require.resolve('@svgr/webpack'),
+                        options: {
+                            svgo: true,
+                            titleProp: true,
+                            ref: true,
+                        },
+                    },
+                    {
+                        loader: require.resolve('url-loader'),
+                        options: {
+                            limit: 10000, // 10kB
+                            name: '[name].[hash:7].[ext]',
+                        },
+                    }],
                 });
             }
             return userWebpack(config, options);
