diff --git a/node_modules/nx/bin/nx.js b/node_modules/nx/bin/nx.js
index f9b3be0..038434f 100755
--- a/node_modules/nx/bin/nx.js
+++ b/node_modules/nx/bin/nx.js
@@ -42,7 +42,7 @@ function main() {
         // Angular CLI, and prettier both use ESM so we need to disable it in these cases.
         if (workspace &&
             workspace.type === 'nx' &&
-            !['format', 'format:check', 'format:write', 'g', 'generate'].some((cmd) => process.argv[3] === cmd)) {
+            !['format', 'format:check', 'format:write', 'g', 'generate'].some((cmd) => process.argv[2] === cmd)) {
             require('v8-compile-cache');
         }
         // polyfill rxjs observable to avoid issues with multiple version of Observable installed in node_modules
