diff --git a/node_modules/nx/bin/nx.js b/node_modules/nx/bin/nx.js
index f9b3be0..038434f 100755
--- a/node_modules/nx/bin/nx.js
+++ b/node_modules/nx/bin/nx.js
@@ -42,7 +42,7 @@ function main() {
         // Angular CLI, and prettier both use ESM so we need to disable it in these cases.
         if (workspace &&
             workspace.type === 'nx' &&
-            !['format', 'format:check', 'format:write', 'g', 'generate'].some((cmd) => process.argv[3] === cmd)) {
+            !['format', 'format:check', 'format:write', 'g', 'generate'].some((cmd) => process.argv[2] === cmd)) {
             require('v8-compile-cache');
         }
         // polyfill rxjs observable to avoid issues with multiple version of Observable installed in node_modules
diff --git a/node_modules/nx/src/command-line/format/format.js b/node_modules/nx/src/command-line/format/format.js
index 4691116..8f94d10 100644
--- a/node_modules/nx/src/command-line/format/format.js
+++ b/node_modules/nx/src/command-line/format/format.js
@@ -51,8 +51,8 @@ async function getPatterns(args) {
             return getPatternsFromProjects(args.projects, graph);
         }
         const p = (0, command_line_utils_1.parseFiles)(args);
-        const supportedExtensions = prettier
-            .getSupportInfo()
+        const supportedExtensions = (await prettier
+            .getSupportInfo())
             .languages.flatMap((language) => language.extensions)
             .filter((extension) => !!extension)
             // Prettier supports ".swcrc" as a file instead of an extension
